extensions/ql-vscode/src/extension.ts

```diff
@@ -480,6 +480,23 @@ async function activateWithInstalledDistribution(
     }
   }
 
+  async function openReferencedFile(
+    selectedQuery: Uri
+  ): Promise<void> {
+    if (qs !== undefined) {
+      if (await cliServer.supportsResolveQlref()) {
+        const resolved = await cliServer.resolveQlref(selectedQuery.path);
+        const uri = Uri.file(resolved.resolvedPath);
+        await window.showTextDocument(uri, { preview: false });
+      } else {
+        helpers.showAndLogErrorMessage(
+          'Jumping from a .qlref file to the .ql file it references is not '
+          + 'supported with the CLI version you are running.\n'
+          + 'Please upgrade your CLI to use this feature.');
```

Comment:
Would be nice to include the minimum cli version here. Something like:

```suggestion
          + `Please upgrade your CLI to version ${CLI_VERSION_WITH_RESOLVE_QLREF} or later to use this feature.`);
```